File: day01/cpp/day01.cpp
#include <iostream>


using namespace std;

// compilar g++ day01.cpp -o day01.exe
int calcYear(int year){

    int days = year*365;
    return days;

}


int main(int argc, char const *argv[])
{
    int year;
    cout << "Qual o ano" << endl;
    cin >> year;
    cout << calcYear(year) << endl;
    return 0;
}

File: day01/java/day01.java
import java.io.IOException;

class day01 {

    public static int calcYear(int year){
        return year*365;
    }

    public static void main(String args[]) throws IOException {
        System.out.println("Qual o ano? ");  
        int year = System.in.read();

        System.out.printf("Ano: %d Dias: %d \n",year, calcYear(year));
    }  
}  

// C:\ProgramData\Oracle\Java\javapath
// C:\Program Files (x86)\Common Files\Oracle\Java\javapath
// javac day01.java
// java day01